mix.exs

```diff
@@ -46,7 +46,7 @@ defmodule AbsintheRelay.Mixfile do
   defp deps do
     [
       {:absinthe, "~> 1.5.0-alpha.0"},
-      {:ecto, "~> 2.0", optional: true},
+      {:ecto, ">= 2.0.0", optional: true},
```

Comment:
This range is too large. I have no way of knowing if there will be an Ecto 4 and if Absinthe.Relay will support it. If it works on 3.0 then we can do `"~> 2.0 or ~> 3.0"`

Reply:
You are very right. Fixed!